frontend/packages/console-plugin-sdk/src/typings/dashboards.ts

```diff
@@ -0,0 +1,51 @@
+import { Extension } from '.';
+import { SubsystemHealth } from '@console/internal/components/dashboards-page/overview-dashboard/health-card';
+
+namespace ExtensionProperties {
+  interface DashboardsOverviewHealthSubsystem<R> {
+    // The subsystem's display name
```

Comment:
Please use `/** comment */` JSDoc syntax.

For brevity, the comment can be written on a single line (unless it's too long):

```ts
/** The subsystem's display name. */
```

This also applies to code below.

Reply:
using JSDoc syntax everywhere now